pyinfra/facts/git.py

```diff
@@ -32,3 +34,21 @@ def process(output):
             items[key] = value
 
         return items
+
+
+class GitTrackingBranch(FactBase):
+    requires_command = 'git'
+
+    @staticmethod
+    def command(repo):
+        return r'cd {0} && git status -sb'.format(repo)
```

Comment:
When parsing git output, it's always better to use the porcelain format to have a well-formatted output.
Even if the Porcelain Format Version 2 is easier to parse, it may be too recent (git 2.11.4, Sept 2017). So we keep using version 1.
Also, short options are great when you are in your shell, but in code, always use the long version. It's more descriptive.

```suggestion
        return 'cd {0} && git status --branch --porcelain'.format(repo)
```